modules/troubleshooting-cluster-upgrader.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting_upgrader_{context}"]
+= Troubleshooting Cluster Upgrader
+
+[role="_abstract"]
+This section describes potential problems and their solutions when using legacy installation method for the secured cluster and enabling the automated updates.
+The errors described here can be found in the clusters view when the upgrader fails.
+
+== Upgrader is missing permissions
+
+.Symptom
+
+The following error is displayed in the cluster page:
+
+[quote]
+____
+Upgrader failed to execute PreflightStage of the roll-forward workflow: executing stage "Run preflight checks": preflight check "Kubernetes authorization" reported errors. This usually means that access is denied. Have you configured this Secured Cluster for automatically receiving upgrades?"
+____
+
+.Procedure
+
+* Ensure that the bundle for Secured Cluster was generated with enabled option for future upgrades: "_Cluster is configured to allow future automatic upgrades_" before clicking on "_Download YAML file and keys_".
```

Comment:
```suggestion
* Ensure that the bundle for Secured Cluster was generated with future upgrades enabled: "_Cluster is configured to allow future automatic upgrades_" before clicking on "_Download YAML file and keys_".
```
I can't put my finger on what's with this sentence, but it feels a bit rough. My rendition doesn't feel much better, but maybe we can find something :sweat_smile: